tools/icu/README.md

```diff
@@ -22,7 +22,7 @@ internationalization functionality.
 
 - Make sure your node workspace is clean (clean `git status`) should be
```

Comment:
Is this intentional? Perhaps minor rephrase would help.

Reply:
Also, «Verify the node build works:» below.

Reply:
The same applies to: `Now, do a clean rebuild of node to test`